### PR TITLE
Aave v3 leverage strategy extension

### PR DESCRIPTION
**Pull request by ckoopmann**

### CODE REVIEW

**Comment by pblivin0x** [comment]
LGTM to start audit 👍

File: test/integration/ethereum/aaveV3LeverageStrategyExtension.spec.ts [hunk 1]
```diff
@@ -0,0 +1,5064 @@
...
+      // const feeRecipient = owner.address;
+      // const maxStreamingFeePercentage = ether(0.1);
+      // const streamingFeePercentage = ether(0.02);
+      // const streamingFeeSettings = {
+      //   feeRecipient,
+      //   maxStreamingFeePercentage,
+      //   streamingFeePercentage,
+      //   lastStreamingFeeTimestamp: ZERO,
+      // };
+      // await .streamingFeeModule.initialize(setToken.address, streamingFeeSettings);
```
Comment (FlattestWhite) [inline test/integration/ethereum/aaveV3LeverageStrategyExtension.spec.ts:315]: remove commented out code?

File: test/integration/ethereum/stakeWiseReinvestmentExtensionIntegration.spec.ts [hunk 2]
```diff
@@ -38,6 +39,8 @@ if (process.env.INTEGRATIONTEST) {
     let extension: StakeWiseReinvestmentExtension;
     let snapshotId: number;
 
+
+  setBlockNumber(16180859);
```
Comment (FlattestWhite) [inline test/integration/ethereum/stakeWiseReinvestmentExtensionIntegration.spec.ts:43]: alignment issue here?

File: utils/constants.ts [hunk 3]
```diff
@@ -22,3 +22,26 @@ export const ONE_MONTH_IN_SECONDS = ONE_DAY_IN_SECONDS.mul(30);
 export const ONE_YEAR_IN_SECONDS = BigNumber.from(31557600);
 
 export const PRECISE_UNIT: BigNumber = constants.WeiPerEther;
+
+export const polygonForkingConfig = {
```
Comment (FlattestWhite) [inline utils/constants.ts:26]: this is duplicated in utils/config.ts?

**Review by FlattestWhite — APPROVED** [review]

File: test/integration/ethereum/stakeWiseReinvestmentExtensionIntegration.spec.ts (hunk 2 above)
Comment (ckoopmann) [reply test/integration/ethereum/stakeWiseReinvestmentExtensionIntegration.spec.ts:43]: Fixed, thanks

File: test/integration/ethereum/aaveV3LeverageStrategyExtension.spec.ts (hunk 1 above)
Comment (ckoopmann) [reply test/integration/ethereum/aaveV3LeverageStrategyExtension.spec.ts:315]: Good catch. Removed it

File: utils/constants.ts (hunk 3 above)
Comment (ckoopmann) [reply utils/constants.ts:26]: It is. Removed it here.

**Pull request merged**